.storybook/main.js

```diff
@@ -63,9 +62,10 @@ module.exports = {
               // sourceMap: argv.sourcemap,
             },
           },
-        ],
-      },
-    ]);
+        ].filter(Boolean),
+        sideEffects: true,
```

Comment:
This is because we are setting the innerHTML?

I had a look for other workarounds in [other react modal storybooks](https://storybook.js.org/showcase/tag/modal/react). Most are just broken. But the [carbon (IBM's design system) modals](https://react.carbondesignsystem.com/?path=/story/components-composedmodal--default) seem to work around it with some sort of manager:

https://github.com/carbon-design-system/carbon/tree/36685a0be5e6/packages/react/src/components/ComposedModal

I don't completely understand it, but maybe it's useful.

Reply:
taken it from CRA sass setup, but I think it might not be needed. It was a "throw stuff at the wall and see what sticks" thing. I'll revert it and see if it breaks anything.

I'll take a look at the Carbon setup.

Reply:
The Twilio one has the same issue at least :stuck_out_tongue: 

![image](https://user-images.githubusercontent.com/5518550/223744188-59046bac-8111-462d-9ae5-28a23cab6e8e.png)

my eyes...